package.json

```diff
@@ -50,6 +50,7 @@
     "fastify-plugin": "^3.0.1",
     "got": "^11.8.5",
     "is-url": "^1.2.4",
+    "iterpal": "^0.3.0",
```

Comment:
[I made this library.](https://github.com/EvanHahn/iterpal) That could be good (I'll do maintenance if there are problems) or bad (it's Evan's unproven module). Happy to rework this without installing a new dependency if we wish.

Reply:
I've been working on the library since 2019 but it has very limited use outside of a couple of (abandoned) side projects. If it were a random person's dependency, I wouldn't include it, but I'd probably be fine if it were a DD developer.

I'll move forward with its inclusion but we can remove it at any time.